src/screens/SettingsScreen.tsx

```diff
@@ -25,9 +25,9 @@ export const SettingsScreen: React.FC<
 
     const dispatch = useAppDispatch();
 
-    const { deg } = useAppSelector(state => state.deg);
-    const { notice } = useAppSelector(state => state.notice);
-    const { theme } = useAppSelector(state => state.theme);
+    const { deg } = useAppSelector(state => state.settings);
+    const { notice } = useAppSelector(state => state.settings);
+    const { theme } = useAppSelector(state => state.settings);
```

Comment:
You can write this in one line with only one destruction like `{ deg, notice, theme }`